hibike/devices/YogiBear/motor.h

```diff
@@ -0,0 +1,23 @@
+#ifndef MOTOR_H
+#define MOTOR_H
+
+//function prototypes
+void motorSetup();
+
+int readEnable();
+
+void enable();
```

Comment:
It would really help if these functions were named `motorEnable`/`motorDisable` or similar. That way when they're called from `Yogibear.cpp` it would be a lot more obvious which file they were coming from.

In fact, given that yogibear is made up of a number of files, it would be helpful if each subsystem prefixed the public functions it exposed. (You can also use namespaces or classes for this, though just naming the functions is fine too)